PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Core/Enum/SpecialitiesOfDoctor.cs

```diff
@@ -0,0 +1,16 @@
+namespace PrivateHospitals.Core.Enum;
+
+public enum SpecialitiesOfDoctor
```

Comment:
```suggestion
public enum DoctorSpecialities
```

We usually don't use word `Of` in naming

Reply:
Fixed